src/components/Loader/Loader.scss

```diff
@@ -1,6 +1,9 @@
 .loader-component {
     display: inline-block;
-    position: relative;
+    position: absolute;
```

Comment:
you shouldnt set absolute position to component. Instead wrap it into div wherever you want to use it, and set absolute position to div